js.go

```diff
@@ -67,9 +67,11 @@ type JetStream interface {
 	// gone.
 	// * If Durable() option is specified, the library will attempt to lookup a JetStream
 	// consumer with this name, and if found, will bind to it and not attempt to
-	// delete it. However, if not found, the library will send a request to create
-	// such durable JetStream consumer. The library will delete the JetStream consumer
-	// after an Unsubscribe() or Drain().
+	// delete it. However, if not found, the library will send a request to
+	// create such durable JetStream consumer. Note that the library will delete
+	// the JetStream consumer after an Unsubscribe() or Drain() *only* if it
```

Comment:
Not sure about additional formatting in the comment, the formatting will not be used or visible in godoc.
```suggestion
	// the JetStream consumer after an Unsubscribe() or Drain() only if it
```